### PR TITLE
XW-1216 | Unify labels for events related with categories menu under the article

### PR DESCRIPTION
**Pull request by bognix**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1216
* https://wikia-inc.atlassian.net/browse/XW-1232

## Description
Unify labels for events related with categories menu under the article. I spotted some not reachable code and I decided to remove it (more details here - https://wikia-inc.atlassian.net/browse/XW-1232)

## Reviewers
@Wikia/x-wing

### CODE REVIEW

File: front/main/app/templates/category.hbs [hunk 1]
```diff
@@ -38,27 +38,6 @@
 			setupAds=true
 			loadBatch=(action 'loadBatch')
 		}}
-
-		<section class="article-footer">
```
Comment (bognix) [inline front/main/app/templates/category.hbs]: This part of code was never used
more context here - https://wikia-inc.atlassian.net/browse/XW-1232

**Comment by vforge** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Pull request merged**